packages/aws-cdk-lib/aws-codedeploy/lib/private/utils.ts

```diff
@@ -31,12 +31,16 @@ export function arnForDeploymentConfig(name: string, resource?: IResource): stri
   });
 }
 
-export function renderAlarmConfiguration(alarms: cloudwatch.IAlarm[], ignorePollAlarmFailure: boolean | undefined, removeAlarms = true):
-CfnDeploymentGroup.AlarmConfigurationProperty | undefined {
+export function renderAlarmConfiguration(
+  alarms: cloudwatch.IAlarm[],
+  ignorePollAlarmFailure?: boolean,
```

Comment:
Hey, why is this optional now?

Reply:
It was possible to be undefined before. Correct me if I'm wrong, but isn't this equivalent? It got called out in community review to simplify, but I'd be happy to revert.

Reply:
It is equivalent. Ideally this would have been part of a property bag to begin with, but now doing so is a breaking change. So no update needed here.